### DISCUSSION TITLE
Version 3.5.7 conflict between tryTo and retryFailedStep

### DISCUSSION THREAD

**Discussion by ZhangXuedan** [codeceptjs/CodeceptJS#3988] [Q&A] [answered]
![image](https://github.com/codeceptjs/CodeceptJS/assets/20330132/c10abc4f-4d3b-470a-a7f9-3c0e5d5f7b0e)
this is my setting

When I use version 3.5.0
![image](https://github.com/codeceptjs/CodeceptJS/assets/20330132/67a82185-196c-4fff-97b0-340a479433f8)

![image](https://github.com/codeceptjs/CodeceptJS/assets/20330132/dddb1c56-1602-46d2-863f-4bed8c29228b)

retryFailedStep plugin to disable steps within a block

When I use version 3.5.7
![image](https://github.com/codeceptjs/CodeceptJS/assets/20330132/6214d9da-fd14-4840-bb12-3594d7d776a9)

![image](https://github.com/codeceptjs/CodeceptJS/assets/20330132/cb9cb4df-f013-459f-806b-f11c48a86397)

retryFailedStep plugin is not disabled within the block

Is this the correct update?

**Comment by adudyak** [comment]
I think in latest versions these plugins enabled automatically, that's why you don't see issue in 3.5.7.
try add option `npx codeceptjs run --plugins retryFailedStep` in 3.5.0.
https://codecept.io/plugins/#retryfailedstep

**Reply by ZhangXuedan** [reply]
In the latest version, is there a way to block the retryFailedStep function in the tryTo code block?
![image](https://github.com/codeceptjs/CodeceptJS/assets/20330132/574e6e86-5b80-468f-b673-56a5aa55dc32)
I think this is a good feature, but I don't know how to set it up in the latest version, retryFailedStep works fine in Tryto block

**Reply by adudyak** [reply]
seems like a bug, it should be disabled. can you show your tryTo portion of code?

**Reply by ZhangXuedan** [reply]
this is my tryTo portion of code. when I use Playwright for web testing and Appium for app testing, I found the same problem in the latest version of CodeceptJS
![image](https://github.com/codeceptjs/CodeceptJS/assets/20330132/170d8e47-554a-45e8-96e9-27f371d9f103)

**Reply by adudyak** [reply, marked as answer]
please report an issue.
As workaround, I can suggest you to use `grabNumberOfVisibleElements` instead of `tryTo`
https://codecept.io/helpers/Playwright/#grabnumberofvisibleelements

**Reply by ZhangXuedan** [reply]
Thanks for the suggestion, I think it's a good alternative. 
This is the issue I reported
https://github.com/codeceptjs/CodeceptJS/issues/4021
